### PR TITLE
LDAP Configure Page Component

### PR DESCRIPTION
**Pull request by zofskeez**
This PR adds a page component for the ldap configure route that includes the form. 

https://github.com/hashicorp/vault/assets/24611656/460ced55-4f2f-47cc-b7cd-ba780517c140

### CODE REVIEW

File: ui/tests/integration/components/ldap/page/configure-test.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          owner: this.engine,
+        }
+      );
+    };
+    this.transitionStub = sinon.stub(this.owner.lookup('service:router'), 'transitionTo');
+  });
+
+  test('it should render empty state when schema is not selected', async function (assert) {
+    await this.renderComponent();
+    // await this.pauseTest();
```
Comment (kiannaquach) [inline ui/tests/integration/components/ldap/page/configure-test.js:68]: ✂️

File: ui/tests/integration/components/ldap/page/configure-test.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    );
+  });
+
+  test('it should populate fields when editing form', async function (assert) {
+    this.model = this.editModel;
+
+    await this.renderComponent();
+
+    assert
+      .dom('[data-test-radio-card="OpenLDAP"]')
```
Comment (kiannaquach) [inline ui/tests/integration/components/ldap/page/configure-test.js:142]: nit: can we use the `selectors.radioCard` here? :)

File: ui/lib/ldap/addon/components/page/configure.hbs [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    <footer class="modal-card-foot modal-card-foot-outlined">
+      <button
+        data-test-save-with-rotate
+        type="button"
+        class="button is-primary"
+        {{on "click" (fn (perform this.save) null true)}}
+      >
+        Save and rotate
+      </button>
+      <button data-test-save-without-rotate type="button" class="button" {{on "click" (fn (perform this.save) null false)}}>
```
Comment (kiannaquach) [inline ui/lib/ldap/addon/components/page/configure.hbs:108]: just out of curiosity why does the event need to be `null`?

File: ui/tests/integration/components/ldap/page/configure-test.js (hunk 1 above)
Comment (zofskeez) [reply ui/tests/integration/components/ldap/page/configure-test.js:68]: 😊

File: ui/tests/integration/components/ldap/page/configure-test.js (hunk 2 above)
Comment (zofskeez) [reply ui/tests/integration/components/ldap/page/configure-test.js:142]: Definitely. Good eye!

File: ui/lib/ldap/addon/components/page/configure.hbs (hunk 3 above)
Comment (kiannaquach) [reply ui/lib/ldap/addon/components/page/configure.hbs:108]: 🤯 that makes sense! learned something new today :)

**Pull request merged**